src/topology/uniform_space/cauchy.lean

```diff
@@ -330,3 +344,119 @@ instance complete_of_compact {α : Type u} [uniform_space α] [compact_space α]
 lemma compact_of_totally_bounded_is_closed [complete_space α] {s : set α}
   (ht : totally_bounded s) (hc : is_closed s) : compact s :=
 (@compact_iff_totally_bounded_complete α _ s).2 ⟨ht, is_complete_of_is_closed hc⟩
+
+/-! *** Sequentially complete space
+
+In this section we prove that a uniform space is complete provided that it is sequentially complete
+(i.e., any Cauchy sequence converges) and its uniformity filter admits a countable generating set.
+In particular, this applies to (e)metric spaces, see file `topology/metric_space/cau_seq_filter`.
+
+More precisely, we assume that there is a sequence of entourages $$U_n$$ such that any other
```

Comment:
Are those double dollars what the new doc website markdown requires?

Reply:
No, I'll fix this in a few minutes. I hope, some day docs will support MathJax in some form.

Reply:
There's no TeX support of any kind in the current doc display. MathJax would probably be the easiest to set up, and would use this syntax. Other options could probably be compatible with this. So I'd say it's future-safe, but won't render as anything special right now.

Reply:
With MathJax, I think single dollar signs would be most appropriate for inline LaTeX. (I didn't see this thread while I was posting my other comment.)

Reply:
Doesn't that increase the risk of false positives? In any case, if and when this is supported, it will be way easier to change `$$` to `$` than vice versa.

Reply:
For the record, I won't get around to adding this to the doc output in the near future. If anyone wants to try, they're welcome to. The script uses the Python `markdown2` library for parsing, which should support arbitrary regex matches that could be used to identify `$$ $$` blocks. I don't know the details but I think it should be easy to set up.

Reply:
Yes, you're right. According to [the docs](http://docs.mathjax.org/en/latest/basic/mathematics.html#tex-and-latex-input), the single dollar signs are actually off by default, in favor of `\(` and `\)` for inline delimiters.

Reply:
For now I replaced dollar signs with `s